ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-presubmits.yaml

```diff
@@ -607,6 +607,62 @@ presubmits:
           name: prow-job-cluster-launch-e2e-azure
         name: job-definition
     trigger: '(?m)^/test (?:.*? )?scaleupdown(?: .*?)?$'
+  - agent: kubernetes
+    always_run: false
```

Comment:
Not sure about this but maybe  `always_run: true` ? Honestly don't know if this has anything to do in periodic